src/core/lib/event_engine/posix_engine/posix_engine_listener.h

```diff
@@ -148,6 +164,7 @@ class PosixEngineListenerImpl
     }
 
    private:
+    PosixListenerWithFdSupport::OnPosixBindNewFdCallback on_append_ = nullptr;
```

Comment:
AnyInvocable's default constructor leaves it in an empty state already.

Reply:
Fixed.